src/port/port.tcl

```diff
@@ -4591,6 +4598,7 @@ proc parse_options { action ui_options_name global_options_name } {
 
         advance
     }
+    set ui_options(options_info_order) $options_info_order
```

Comment:
Not really sure this fits into `ui_options`, as everything else related to an action is in `global_options`. Or am I missing something?